Sources/XCHammer/XcodeTarget.swift

```diff
@@ -25,28 +25,19 @@ private func shouldPropagateDeps(forTarget xcodeTarget: XcodeTarget) -> Bool {
 }
 
 /// Return XCConfig files
-/// Currently we support `Diags.xcconfig`, the base for all
-/// warning flags.
-private func getDiagsXCConfigFiles(for target: XcodeTarget, genOptions: XCHammerGenerateOptions) -> [String: String] {
-    // Linear search a sequence of Maybe XCConfigs for the first XCConfig
-    // Bias towards a Diags.xcconfig closest to the BUILD file
```

Comment:
I think this logic is still nice to have in a way - otherwise, it can be cumbersome to create multiple `XCHammer.yaml` config files. We may want to consider adding this back in the future - automatically picking up `.xcconfig` files adjacent to the `BUILD` file